polynote-frontend/polynote/data/codec.test.ts

```diff
@@ -35,6 +35,8 @@ describe("A Codec", () => {
         expect(Codec.decode(codec, Codec.encode(codec, value))).toEqual(value)
     }
 
+    const safeUnicode = () => fc.unicode().filter(char => char !== "﻿") // Zero-width non-breaking space doesn't roundtrip through the TextEncoder.
```

Comment:
This is not an empty string, it's actually a [zero-width non-breaking space (a.k.a. Byte-order mark)](https://en.wikipedia.org/wiki/Byte_order_mark) which TextDecoder doesn't handle well.